modules/core/shared/src/main/scala-3/io/circe/derivation/ConfiguredEncoder.scala

```diff
@@ -55,19 +55,21 @@ trait ConfiguredEncoder[A](using conf: Configuration) extends Encoder.AsObject[A
 
 object ConfiguredEncoder:
   inline final def derived[A](using conf: Configuration)(using inline mirror: Mirror.Of[A]): ConfiguredEncoder[A] =
-    new ConfiguredEncoder[A] with SumOrProduct:
-      lazy val elemLabels: List[String] = summonLabels[mirror.MirroredElemLabels]
-      lazy val elemEncoders: List[Encoder[?]] = summonEncoders[mirror.MirroredElemTypes]
+    given result: ConfiguredEncoder[A] =
```

Comment:
FYI: note that I only added this line and line 72 - anything in between is just an additional layer of indentation.